TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Command/NodeCommandControllerPlugin.php

```diff
@@ -1005,9 +1005,11 @@ protected function collectNodesWithInvalidDimensions($workspaceName, array $allo
             foreach ($allowedDimensionCombinations as $dimensionConfiguration) {
                 ksort($dimensionConfiguration);
                 ksort($nodeDataArray['dimensionValues']);
-                if ($dimensionConfiguration === $nodeDataArray['dimensionValues']) {
-                    $foundValidDimensionValues = true;
-                    break;
+                foreach ($dimensionConfiguration as $dimensionKey => $dimensionValuesArray) {
+                    if (in_array($nodeDataArray['dimensionValues'][$dimensionKey][0], $dimensionValuesArray)) {
```

Comment:
thanks! Is it better to use `isset` or `array_key_exists` in such cases?

Anyways, I'm waiting for @robertlemke to take over on this one, as I'm not sure what would be the fate of this feature in general. For time being I would have just removed it, it's very dangerous to keep it as is!